src/include/homeobject/pg_manager.hpp

```diff
@@ -76,7 +78,7 @@ struct PGStats {
 class PGManager : public Manager< PGError > {
 public:
     virtual NullAsyncResult create_pg(PGInfo&& pg_info) = 0;
-    virtual NullAsyncResult replace_member(pg_id_t id, peer_id_t const& old_member, PGMember const& new_member) = 0;
+    virtual NullAsyncResult replace_member(pg_id_t id, peer_id_t const& old_member, PGMember const& new_member, u_int32_t const& commit_quorum) = 0;
```

Comment:
why do we need const& for commit_qurorum. Simple uint32_t commit_quorum is enough.

Reply:
Okay.